.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**I confirm:**
+<!-- To mark a task as complete, use [x] -->
+- [ ] I haven't found an existing/similar bug report for my bug.
+- [ ] this bug report is about a single actionable bug.
+- [ ] if this issue has been spotted on the git-scm.com site, I've cross-checked that my issue is still present in the current pdf version.
+- [ ] this bug is about the second version of the Git Pro book in English. A bug for a translation should be filed with the translation project. You can find a table with those projects here: [progit2/TRANSLATING.md](https://github.com/progit/progit2/blob/master/TRANSLATING.md)
```

Comment:
normally git-scm and pdfs are updated roughly at the same pace (the webpages and the links to pdfs are updated at the same time)

Reply:
This part:
>- [ ] if this issue has been spotted on the git-scm.com site, I've cross-checked that my issue is still present in the current pdf version.

has been copied from:
https://github.com/progit/progit2/blob/master/CONTRIBUTING.md#signaling-an-issue

So I thought that would be correct/helpful to add into the template.

If nowadays the pdfs are build rightaway when the site updates, this section only adds busy-work for no real value, and we can remove the section from the template (and from the CONTRIBUTING.md file...). So would you like to keep that part, or do you want to get rid of it?

Reply:
This an help to find out if the issue is coming from the source or from the tools.

Reply:
Shouldn't the line be changed? Seeing as the likely source of trouble is in the tools (pdf builder)?

Maybe something like this would be clearer?
```
- [ ] if this issue has been spotted in the pdf, I've checked that the problem is also present on the [website](https://www.git-scm.com/book/en/v2)
```
That would at least fix this particular line.

Reply:
:+1:

Reply:
I've changed the line.

Reply:
the ebook versions are also possible here.

Reply:
This is changed to:

```
**Which version of the book is affected?**
<!-- It's important for us to know if the problem is in the source or in the tooling for the pdf/epub/mobi files. -->
<!-- Therefore, please write whether the problem is with the files, the online book, or both. -->
```

This also explains the reason why we need the comparison between source and the converted files.